src/lib.rs

```diff
@@ -1,4 +1,8 @@
 #[cfg(target_arch = "wasm32")]
+extern crate gameboy_opengl_web;
```

Comment:
Rust 2018 edition makes these `extern crate` optional (unless you need it for `#[macro_use]`), so we can delete

```
#[cfg(target_arch = "wasm32")]
extern crate gameboy_opengl_web;
```

Reply:
Deleted the Extern crate.